packages/framework/ai-collab/src/explicit-strategy/debugEvents.ts

```diff
@@ -103,15 +111,25 @@ export interface GenerateTreeEditCompletedDebugEvent extends EventFlowDebugEvent
 	llmGeneratedEdit?: TreeEdit | null;
 }
 
-// Apply Edit Debug events: ----------------------------------------------------------
+/**
+ * ----------------------------------------------------------------------------------
+ * Apply Edit Debug events
+ * ----------------------------------------------------------------------------------
+ */
 
 /**
  * A debug event marking the successful application of an LLM generated edit to a SharedTree.
  * @alpha
  */
 export interface ApplyEditSuccessDebugEvent extends DebugEvent {
 	eventName: "APPLIED_EDIT_SUCCESS";
+	/**
+	 * A unique id that will be shared across all debug events that are part of the same event flow.
+	 */
```

Comment:
I find it weird that this (and others below) has an `eventFlowTraceId` but is not an `EventFlowDebugEvent`. Is that correct?